hack/build-tests.sh

```diff
@@ -14,7 +14,7 @@ if [ "${JOB_TYPE}" == "travis" ]; then
     go mod vendor
     PACKAGE_PATH="pkg/"
     mkdir -p coverprofiles
-    KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1 ginkgo -r -cover -outputdir=./coverprofiles -coverprofile=cover.coverprofile ${PACKAGE_PATH}
+    KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1 ginkgo -r -covermode atomic -outputdir=./coverprofiles -coverprofile=cover.coverprofile ${PACKAGE_PATH}
```

Comment:
are you sure you don't need to still specify the `cover` flag to run the coverage tests?
```
  -cover
    	Run tests with coverage analysis, will generate coverage profiles with the package name in the current directory.
```

Reply:
It's not needed if we are using the `-covermod` flag

Reply:
/lgtm